cdap-master/src/main/java/co/cask/cdap/master/startup/HBaseCheck.java

```diff
@@ -42,6 +45,14 @@ private HBaseCheck(Configuration hConf) {
 
   @Override
   public void run() {
+    LOG.info("Checking HBase Version");
+    try {
+      new HTableNameConverterFactory().get();
+    } catch (ProvisionException e) {
+      throw new RuntimeException("Unsupported Hbase version " + HBaseVersion.getVersionString());
+    }
+    LOG.info("  HBase Version successfully verified.");
```

Comment:
Can we have all these strings be the same: `HBase version`, instead of a mix of `version` and `Version`?